Lombiq.NodeJs.Extensions.Samples/package.json

```diff
@@ -1,12 +1,9 @@
 {
   "private": true,
   "devDependencies": {
-    "nodejs-extensions": "link:..\\Lombiq.NodeJs.Extensions",
     "npm-run-all": "^4.1.5"
   },
   "scripts": {
-    "dotnet-prebuild": "pnpm build",
-    "dotnet-postclean": "pnpm clean",
```

Comment:
@Piedone This has moved into the main project, now.

Reply:
Great!